conftest.py

```diff
@@ -0,0 +1,142 @@
+from django.contrib.auth import get_user_model
+from player.models import BallGame, Player
+from player_rating.models import PlayerRating, Rating
+from court.models import Court
+from game_event.models import GameEvent
+from decimal import Decimal
+from django.utils import timezone
+from message.models import Message
+from game_event_player.models import GameEventPlayer
+from court_ball_game.models import CourtBallGame
+import pytest
+
+TEST_ID = 2
+TEST_TIME = timezone.now()
+TEST_LEVEL = 3
+TEST_MIN = 2
+TEST_MAX = 5
+TEST_BALL_GAME = 'Basketball'
+
+ID1 = 2
+ID2 = 3
```

Comment:
I don't think it makes sense to have constants defined here. conftest won't have them imported automatically, so you now need to maintain them in multiple places, and suppose that you get them imported manually, anyone changing any of them in one test (they are only constants by convention - they can actually be changed), they are now changed to all other tests - potentially breaking them.

For the most part, it's best to either define stuff as fixtures (in that case they are recreated for each test, or only use them directly only at one place (e.g. when you create an instance).

```python
@pytest.fixture
def foo():
    return Foo(bar=2)

def test_foo_does_something(foo):
    old_bar  = foo.bar
    output = func_that_returns_bar_divided_by_3(foo)
    assert output = old_bar / 3
```

Reply:
Removed the constatns.